### PR TITLE
TASK: fix deleting problem for web Link

### PR DESCRIPTION
**Pull request by FahimNsr**

### CODE REVIEW

**Comment by andrehoffmann30** [comment]
Hi @FahimNsr I miss the compiled plugin code in your PR. You need the commit the compiled plugin code otherwise your changes will not be use, when this package is installed in a project. Please the missing code changes. Thank you.

File: Neos.Ui/core/src/application/LinkTypes/Web/Web.tsx [hunk 1]
```diff
@@ -75,40 +82,39 @@ export const Web = makeLinkType<WebLinkModel>('Sitegeist.Archaeopteryx:Web', ({c
...
+                                    value: 'https',
+                                    label: 'HTTPS',
+                                    icon: 'lock'
+                                }, {
+                                    value: 'http',
+                                    label: 'HTTP',
+                                    icon: 'unlock'
+                                }]}
+                            />
+                            {input.value !== undefined && setProtocol(input.value)}
```
Comment (andrehoffmann30) [inline Neos.Ui/core/src/application/LinkTypes/Web/Web.tsx]: This is not the right place to change the State. You do not change the State on your component rendering. Please adjust the State in the `onValeChange` function. This kind of code is also unclear and miss leading. Please do not use this code style.

**Review by andrehoffmann30 — CHANGES_REQUESTED** [review]

File: Neos.Ui/core/src/application/LinkTypes/Web/Web.tsx (hunk 1 above)
Comment (FahimNsr) [reply Neos.Ui/core/src/application/LinkTypes/Web/Web.tsx]: I tried to do that in onValeChange scope but, I could not get the value there.

**Comment by FahimNsr** [comment]
Hi @andrehoffmann30, I added the compiled file.

**Review by andrehoffmann30 — APPROVED** [review]

**Pull request merged**